docs/source/features/tool_calling.md

```diff
@@ -1,6 +1,6 @@
 # Tool Calling
 
-vLLM currently supports named function calling, as well as the `auto` and `none` options for the `tool_choice` field in the chat completion API. The `tool_choice` option `required` is **not yet supported** but [on the roadmap](gh-issue:13002).
+vLLM currently supports named function calling, as well as the `auto`, `required` and `none` options for the `tool_choice` field in the chat completion API.
```

Comment:
Probably need to add a version here to indicate when this is supported

Reply:
Added a (placeholder) version to the required option.